docs/upgrade-to-v5.md

```diff
@@ -0,0 +1,37 @@
+# Changes in v5
+
+## About
+
+The following is a detailed collection of the changes in the major v5 release of the bson package
+for nodejs and web platforms.
+
+<!--
+1. a brief statement of what is breaking (brief as in "x will now return y instead of z", or "x is no longer supported, use y instead", etc
+2. a brief statement of why we are breaking it (bug, not useful, inconsistent behavior, better alternative, etc)
+3. if applicable, an example of suggested syntax change (can be included in (1) )
+-->
+
+### Remove reliance on Node.js Buffer
+
+> **TL;DR**: Web environments return Uint8Array; Node.js environments return Buffer
+
+For those that use the BSON library on Node.js, there is no change the BSON APIs will still return and accept instances of Node.js Buffer. Since we no longer depend on the Buffer web shim for compatibility with browsers, in non-Node.js environments a Uint8Array will be returned instead.
```

Comment:
```suggestion
For those that use the BSON library on Node.js, there is no change - the BSON APIs will still return and accept instances of Node.js Buffer. Since we no longer depend on the Buffer web shim for compatibility with browsers, in non-Node.js environments a Uint8Array will be returned instead.
```

Reply:
done!